src/app/statistic/statistic.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'sh-statistic',
```

Comment:
Like sh-devices-list or sh-widgets-list or something similar. You can view statistic only for sensors. For servo and toggles you can't.